casper/src/test/scala/coop/rchain/casper/util/DagOperationsTest.scala

```diff
@@ -40,20 +44,36 @@ class DagOperationsTest
         for {
           genesis <- createBlock[Task](Seq.empty)
           b1      <- createBlock[Task](Seq(genesis.blockHash))
+          b1m     = BlockMetadata.fromBlock(b1)
```

Comment:
For less repetition and improved readability you could use a helper function which returns a case class containing the block and its metadata. 

I'm thinking something like this:

```scala
final case class BlockWithMeta(val block, val metadata)

def createBlockWithMeta(...) = {...}

for {
 b1 <- createBlockWithMeta(...)
 b2 <- createBlockWithMeta(b1.block.blockHash)
 ...
_ <- DagOperations.lowestCommonAncestorF[Task](b1.metadata, b5.metadata, dag) shouldBeF b1.metadata
}
```